.github/workflows/pre-commit.yml

```diff
@@ -0,0 +1,14 @@
+name: pre-commit
+
+on:
+  push:
+    branches: [main]
+  pull_request:
+
+jobs:
+  pre-commit:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
+    - uses: actions/setup-python@v3
+    - uses: pre-commit/action@v3.0.0
```

Comment:
We already run pre-commit via it's CI offerings. I don't think we need this ?

Reply:
Aha yes, you're right, I forgot that pre-commit had their own CI integration.

I've dropped the commit that added this file.